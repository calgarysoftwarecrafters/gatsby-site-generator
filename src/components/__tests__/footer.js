import React from "react"
import renderer from "react-test-renderer"
import Footer from "../footer"
describe("Footer", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Header siteTitle="Default Starter" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
