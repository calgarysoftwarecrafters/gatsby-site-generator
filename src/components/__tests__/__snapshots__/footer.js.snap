// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<div
  className="footer"
>
  <div
    className="container"
  >
    <hr
      className="border-primary"
    />
    <p>
      <a
        href="/about/"
      >
        <br />
        <strong />
         on About
      </a>
    </p>
  </div>
</div>
`;
